anthos-bm-gcp-terraform/main.tf

```diff
@@ -117,7 +117,7 @@ module "instance_template" {
     enable-oslogin = "false"
   }
   service_account = {
-    email  = ""
+    email  = var.service_account  # service account used for GCE instances
```

Comment:
```suggestion
    email  = var.gce_vm_service_account
```

Reply:
See [comment below](https://github.com/GoogleCloudPlatform/anthos-samples/pull/120/files#r787953957). Can we have this also changed accordingly